runtimes.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
```

Comment:
Is this code ours? Or is this a copy of TI's code? If so, we should preserve the same TI's demos headers

Reply:
Apply this to all the files with TI code